Software/Python/grove_dht_pro_filter/grove_dht_example1.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+#
+# The GrovePi connects the Raspberry Pi and Grove sensors.  You can learn more about GrovePi here:  http://www.dexterindustries.com/GrovePi
+#
```

Comment:
Can you add "Connect the Grove DHT Sensor (Blue One) << It's the blue one right?>> to Port 4 in this example."